applications/unite-analysis-deseq2/docker-compose.build.yml

```diff
@@ -4,6 +4,12 @@ services:
     extends:
       file: docker-compose.yml
       service: deseq2.analysis.unite.net
+    build:
+      context: ./src
+      dockerfile: ./Dockerfile
+      args:
```

Comment:
I think this dockerfile does really need any arguments to be passed on.

Reply:
Yes, fixed.